Dockerfile

```diff
@@ -147,6 +147,35 @@ COPY --from=build-src /usr/share/licenses/testsys /licenses/testsys
 
 CMD dockerd --storage-driver vfs &>/dev/null & ./vsphere-k8s-cluster-resource-agent
 
+# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
+# Builds the Metal K8s cluster resource agent image
+FROM public.ecr.aws/amazonlinux/amazonlinux:2 as metal-k8s-cluster-resource-agent
+
+RUN yum install -y \
+        openssh-clients \
+        tar \
+        &&  yum clean all
```

Comment:
super nit, but I would probably unindent this as it is not part of that first yum command. (also you have an extra space)